### PR TITLE
Update Command.cs

### PR DESCRIPTION
**Pull request by drwill-ms**

### CODE REVIEW

File: AdtSampleApp/SampleClientApp/CommandLoop.cs [hunk 1]
```diff
@@ -54,10 +53,10 @@ public async Task CommandCreateModels(string[] cmd)
...
                 }
-                Response<ModelData[]> res = await client.CreateModelsAsync(dtdlList);
+                Response<DigitalTwinsModelData[]> res = await client.CreateModelsAsync(dtdlList);
                 Log.Ok($"Model(s) created successfully!");
-                foreach (ModelData md in res.Value)
-                    LogResponse(md.Model);
+                foreach (DigitalTwinsModelData md in res.Value)
+                    LogResponse(md.DtdlModel);
             }
             catch (RequestFailedException e)
```
Comment (bikamani) [inline AdtSampleApp/SampleClientApp/CommandLoop.cs:61]: Add a catch specifically for conflict case?
// If a model already exists
(RequestFailedException e) when (e.Status == (int)HttpStatusCode.Conflict)

Comment (drwill-ms) [reply AdtSampleApp/SampleClientApp/CommandLoop.cs:61]: I'm not updating their sample to be better just to use the new SDK.

File: AdtSampleApp/SampleClientApp/CommandLoop.cs [hunk 2]
```diff
@@ -121,13 +120,18 @@ public async Task CommandGetModels(string[] cmd)
...
+                AsyncPageable<DigitalTwinsModelData> results = client.GetModelsAsync(
+                    new GetModelsOptions
+                    {
+                        DependenciesFor = dependenciesFor,
+                        IncludeModelDefinition = includeModelDefinitions
+                    });
+                var reslist = new List<DigitalTwinsModelData>();
+                await foreach (DigitalTwinsModelData md in results)
                 {
                     Log.Out(md.Id);
```
Comment (bikamani) [inline AdtSampleApp/SampleClientApp/CommandLoop.cs:132]: add more description here?

Retrieved model '{md.Id}'

File: AdtSampleApp/SampleClientApp/CommandLoop.cs [hunk 3]
```diff
@@ -536,14 +534,14 @@ public async Task CommandCreateRelationship(string[] cmd)
             {
                 for (int i = 0; i < args.Length; i += 3)
                 {
-                    relationship.CustomProperties.Add(args[i], ConvertStringToType(args[i + 1], args[i + 2]));
+                    relationship.Properties.Add(args[i], ConvertStringToType(args[i + 1], args[i + 2]));
```
Comment (bikamani) [inline AdtSampleApp/SampleClientApp/CommandLoop.cs:537]: This is hard to read but I understand if it's not in the scope of changes currently.

**Review by bikamani — APPROVED** [review]

File: AdtSampleApp/SampleClientApp/CommandLoop.cs (hunk 2 above)
Comment (drwill-ms) [reply AdtSampleApp/SampleClientApp/CommandLoop.cs:132]: As discussed, just updating to work with the new nuget.

**Pull request merged**